### ISSUE TITLE
Can the runtime files be separated from the source code?

### ISSUE THREAD

**Issue by varyoo** [jfurrow/flood#366]
For example `config.js` or the file that holds the password. In order to package Flood as a regular program.

**Comment by jfurrow** [comment]
Thanks for the issue! Do you mean you'd like to see `config.js` in a separate or configurable directory? Or injected as environment variables?

**Comment by varyoo** [comment]
For now if I want to run two instances of Flood on the same computer I have to clone all the files (source code, configuration and runtime files) for each instance. I was wondering if the source code could be separated from the other two, so it can be used by all instances. Not sure if this is usual or even desirable with Javscript programs. Thank you!

**Comment by Phr33d0m** [comment]
Hello @varyoo , from what I've seen you can wipe the 'client' folder and flood will work just fine.

At first I thought you want to try to package flood for a certain linux distro, although it seems you want an easy way to run multiple instances of flood? I'm currently running 10 instances of flood from the same folder with a very ugly hack to gulpfile which receives the config file as a paremeter. As currently pull-reqs are slowly reviewed I didn't want to overflow the dev with stuff so I've been delaying it until I have a better solution than my current 'hack'.

If that's what you need right now just drop a line and I'll post the file for you, although I prefer a thorough discussion on how this should be made properly with the dev.

**Comment by varyoo** [comment]
Yes I wanted to package it, which require sharing the same code for multiple instances, not a priority I agree.
